### ISSUE TITLE
An already existing user, tries to signup, results in flunked (blank page)

### ISSUE THREAD

**Issue by vijaykanthm28** [megamsys/nilavu#822]
no implicit conversion of Array into String
_flyasm.html.erb:34:in `+'
_flyasm.html.erb:34:in`block in _app_views_cockpits__flyasm_html_erb___1447524783066291909_70091804057200'
_flyasm.html.erb:26:in `each'
_flyasm.html.erb:26:in`_app_views_cockpits__flyasm_html_erb___1447524783066291909_70091804057200'
_flygrid.html.erb:3:in `_app_views_cockpits__flygrid_html_erb__2972531527571228050_70091803994560'
_mycockpit.html.erb:20:in`block in _app_views_cockpits__mycockpit_html_erb__52863745751877001_70091803914020'
_mycockpit.html.erb:16:in `_app_views_cockpits__mycockpit_html_erb__52863745751877001_70091803914020'
index.html.erb:3:in`_app_views_cockpits_index_html_erb___3897640758195260377_70091803892780'
global_exceptions.rb:4:in `catch_exceptions'

```
   ,--.!,
__/   -*-
,####.  '|`
######
`####'                !We flunked!
```

Completed 200 OK in 42ms

**Comment by indykish** [comment]
we are moving to ember. So its easy to do it.

**Comment by rajesh-rajagopal** [comment]
Existing user try to sign up its accepted and try to login its go for maintainence page

Processing by UsersController#create as HTML
  Parameters: {"utf8"=>"✓", "authenticity_token"=>"+H5GfqlSC5Yii8aAKaSxPoUmottPvQMol0dhCIDodRiX58By2Ijur9nJ3xo8x4osvctARVdIm7w6V9sAUWNPiw==", "email"=>"rajram.rajesh21@gmail.com", "password"=>"[FILTERED]"}
FASCADE Megam::Accountcreate 
START Megam::Account.create

> email: rajram.rajesh21@gmail.com
> api_key: 5c466201cae4a17cd7afce29aa96cb8ec1daa60f
> password: cmFqZXNo
> first_name: Rajram Rajesh21
> last_name: 
> host: 88.198.139.80
> RUN_NOW Megam::Account.create 
> Redirected to http://88.198.139.80:8080/cockpits
> Completed 302 Found in 12ms
> Started GET "/cockpits" for 182.74.209.86 at 2016-05-09 17:52:32 +0300
> Processing by CockpitsController#index as HTML
> FASCADE Megam::Organizationslist 
> START Megam::Organizations.list
> email: rajram.rajesh21@gmail.com
> api_key: 5c466201cae4a17cd7afce29aa96cb8ec1daa60f
> org_id: 
> host: 88.198.139.80
> RUN_NOW Megam::Organizations.list 
> Completed 500 Internal Server Error in 6ms

NoMethodError (undefined method `each' for nil:NilClass):
  app/models/teams.rb:16:in`find_all'
  app/controllers/application_controller.rb:99:in `block in set_current_user_with_team'
  app/controllers/application_controller.rb:98:in`tap'
  app/controllers/application_controller.rb:98:in `set_current_user_with_team'

**Comment by indykish** [comment]
Fixed in 1.5
